### ISSUE TITLE
Edit Cards issue, edit button not visable

### ISSUE THREAD

**Issue by pissten** [dwainscheeren/dwains-lovelace-dashboard#630]
After the last two or so, updates, I've had issues with editing cards. When enable edit mode, the menu popup is not visible, making it impossible to press the edit card properties button. Also some text is very unclear, especially in the welcome header (weather entity).

![image](https://user-images.githubusercontent.com/16886605/192090766-09338bb1-ffb5-47c3-bcc8-b915690d6fa0.png)
![image](https://user-images.githubusercontent.com/16886605/192090788-1add475c-9522-4df8-83c3-dd76d7034e9a.png)
![image](https://user-images.githubusercontent.com/16886605/192090838-eae839ed-02f3-44c4-9e9f-538e027de7a6.png)

**Comment by dwainscheeren** [comment]
Is this on a computer in which browser?

**Comment by pissten** [comment]
Hi Dwains,

This is on a computer in chrome browser. But also acts the same on other computers and browsers. Also tested in mobile browser with same results.

Best regards
Christian
________________________________

**Comment by dwainscheeren** [comment]
Can you check in your chrome browser if you got any error in the console log of the developer tools? https://balsamiq.com/support/faqs/browserconsole/#google-chrome

**Comment by pissten** [comment]
Seems like there were some errors.. I've exported the console output to a txt, that I've attached..

________________________________
Fra: Dwain Scheeren ***@***.***>
Sendt: mandag 26. september 2022, 11:47
Til: dwainscheeren/dwains-lovelace-dashboard ***@***.***>
Kopi: pissten ***@***.***>; Author ***@***.***>
Emne: Re: [dwainscheeren/dwains-lovelace-dashboard] Edit Cards issue, edit button not visable (Issue #630)


Can you check in your chrome browser if you got any error in the console log of the developer tools? https://balsamiq.com/support/faqs/browserconsole/#google-chrome

**Comment by dwainscheeren** [comment]
I can't see the txt file.

**Comment by pissten** [comment]
You didn't receive the attachment? Maybe I'll have to zip the file to let it go through the firewall then...

See if there's a zip file attached now

Br
Chris
________________________________
Fra: Dwain Scheeren ***@***.***>
Sendt: mandag 26. september 2022 12:07
Til: dwainscheeren/dwains-lovelace-dashboard ***@***.***>
Kopi: pissten ***@***.***>; Author ***@***.***>
Emne: Re: [dwainscheeren/dwains-lovelace-dashboard] Edit Cards issue, edit button not visable (Issue #630)


I can't see the txt file.

**Comment by dwainscheeren** [comment]
Can you post the content of the txt on a site like https://pastebin.com/ and place the url here maybe

**Comment by pissten** [comment]
https://pastebin.com/MTAU7EWP
[https://pastebin.com/i/facebook.png]<https://pastebin.com/MTAU7EWP>
dwains_theme_browser_log - Pastebin.com<https://pastebin.com/MTAU7EWP>
Pastebin.com is the number one paste tool since 2002. Pastebin is a website where you can store text online for a set period of time.
pastebin.com

________________________________
Fra: Dwain Scheeren ***@***.***>
Sendt: mandag 26. september 2022 12:26
Til: dwainscheeren/dwains-lovelace-dashboard ***@***.***>
Kopi: pissten ***@***.***>; Author ***@***.***>
Emne: Re: [dwainscheeren/dwains-lovelace-dashboard] Edit Cards issue, edit button not visable (Issue #630)


Can you post the content of the txt on a site like https://pastebin.com/ and place the url here maybe

**Comment by pissten** [comment]
Hi Dwain,

Did you receive what you needed? Anything else i could provide to help you in the troubleshooting?


Best Regards
Christian
________________________________
Fra: Christian Hegland ***@***.***>
Sendt: mandag 26. september 2022 12:28
Til: dwainscheeren/dwains-lovelace-dashboard ***@***.***>
Emne: Sv: [dwainscheeren/dwains-lovelace-dashboard] Edit Cards issue, edit button not visable (Issue #630)

https://pastebin.com/MTAU7EWP
[https://pastebin.com/i/facebook.png]<https://pastebin.com/MTAU7EWP>
dwains_theme_browser_log - Pastebin.com<https://pastebin.com/MTAU7EWP>
Pastebin.com is the number one paste tool since 2002. Pastebin is a website where you can store text online for a set period of time.
pastebin.com

________________________________
Fra: Dwain Scheeren ***@***.***>
Sendt: mandag 26. september 2022 12:26
Til: dwainscheeren/dwains-lovelace-dashboard ***@***.***>
Kopi: pissten ***@***.***>; Author ***@***.***>
Emne: Re: [dwainscheeren/dwains-lovelace-dashboard] Edit Cards issue, edit button not visable (Issue #630)


Can you post the content of the txt on a site like https://pastebin.com/ and place the url here maybe

**Comment by pissten** [comment]
![image](https://user-images.githubusercontent.com/16886605/192363651-d153609a-6d63-49b8-a429-40b65962ee4a.png)

**Comment by pissten** [comment]
![image](https://user-images.githubusercontent.com/16886605/192444830-0a92739b-7ed4-401f-949d-44fc360064a5.png)

**Comment by pissten** [comment]
![image](https://user-images.githubusercontent.com/16886605/192444966-054a2de9-f4ed-41e5-8e8d-1aa93a8ba08f.png)

**Comment by dwainscheeren** [comment]
It seems like an error with your HA setup. Can you please check if everything is correct installed.

**Comment by pissten** [comment]
How can I check that? It all worked well untill a later home assistant core update. After upgrade this issue came. I can't simply downgrade, beacuse the zwavejs upgrade solved some other issues i were experienceing earlier 🫤
________________________________

**Comment by pissten** [comment]
hi again Dwain,

So I've tried some things to try to solve this issue, but unfortunately without results. Removed all custom popup cards. Uninstalled browser mod. Removed all custom cards besides the nest thermostat card. Reinstalled dwains. Still not able to use edit mode to change card settings or move cards around. This Is though only a issue in the main page. In the devices page everything is normal :S Any ideas?
